d2renderers/d2fonts/d2fonts.go

```diff
@@ -134,165 +140,196 @@ var fuzzyBubblesBoldBase64 string
 //go:embed ttf/*
 var fontFacesFS embed.FS
 
-var FontEncodings map[Font]string
-var FontFaces map[Font][]byte
+// FontEncodings map[Font]string
+var FontEncodings sync.Map
+
+// FontFaces map[Font][]byte
+var FontFaces sync.Map
```

Comment:
i say we wrap this in a struct with methods so you don't have to cast everywhere you use it.